src/RawPOSIX/src/interface/mem.rs

```diff
@@ -226,3 +230,21 @@ pub fn sbrk_handler(cageid: u64, brk: u32) -> i32 {
 
     (PAGESIZE * heap.npages) as i32
 }
+
```

Comment:
also once we make the modifications a long form comment about how this works and its use would be helpful